src/picknik_ur_base_config/objectives/move_to_joint_state.xml

```diff
@@ -3,7 +3,7 @@
     <BehaviorTree ID="Move to Joint State" _description="Move to a specified joint state">
         <Control ID="Fallback" name="root">
             <Control ID="Sequence">
-                <Action ID="RetrieveJointStateParameter" timeout_sec="-1" joint_state="{target_joint_state}"/>
+                <Action ID="RetrieveJointStateParameter" timeout_sec="1.0" joint_state="{target_joint_state}"/>
```

Comment:
don't change this timeout, it's supposed to block indefinitely

Reply:
This is spamming like crazy in the UI. Are we sure this is blocking indefinitely. Feels like it is not blocking at all

Reply:
I think this may have been a behavior change from @pac48, then -- but -1 is supposed to block indefinitely once fixed.